flooder/src/main.rs

```diff
@@ -1,100 +1,258 @@
-use std::time::Duration;
+use core::cmp::min;
 
 use aleph_client::{
-    account_from_keypair, pallets::balances::BalanceUserApi, raw_keypair_from_string, AccountId,
-    Balance, KeyPair, SignedConnection, SignedConnectionApi, TxStatus,
+    account_from_keypair,
+    pallets::{
+        author::AuthorRpc, balances::BalanceUserApi, system::SystemApi, timestamp::TimestampApi,
+    },
+    raw_keypair_from_string,
+    utility::BlocksApi,
+    AccountId, Balance, KeyPair, Nonce, SignedConnection, SignedConnectionApi, TxStatus, TOKEN,
 };
 use clap::Parser;
 use config::Config;
 use futures::future::join_all;
-use log::info;
-use subxt::ext::sp_core::{sr25519, Pair};
-use tokio::{time, time::sleep};
+use log::{debug, info};
+use subxt::{
+    ext::sp_core::{sr25519, Pair},
+    tx::TxPayload,
+    utils::{MultiAddress, Static},
+};
+use tokio::time::{sleep, Duration, Instant};
 
 mod config;
 
+/// Accounts used for the experiment (except for the main account) will have seed //1 etc.
+const ACCOUNTS_SEED_PREFIX: &str = "//";
```

Comment:
it could be local to function const or even this could be inlined

Reply:
Changed to fn const